### PR TITLE
config: Use environment variables for configuration

### LINKED ISSUE #13: Use environment variables instead of hardcoding it in config file

### PR DESCRIPTION
**Pull request by meetmangukiya**
Closes #13

### CODE REVIEW

**Comment by nemani** [comment]
Approved.

File: config.py [hunk 1]
```diff
@@ -16,11 +16,13 @@
...
-BOT_ADMINS = ('meetmangukiya', )  # !! Don't leave that to "CHANGE ME" if you connect your errbot to a chat system !!
+BOT_ADMINS = (os.environ.get('BOT_ADMIN'), )  # !! Don't leave that to "CHANGE ME" if you connect your errbot to a chat system !!
 
 BOT_IDENTITY = {
-    'token' : '<token>'
+    'token' : os.environ.get("GITTER_TOKEN")
 }
 
+
 CHATROOM_PRESENCE = ('coala/cobot-test', 'coala/errbot') # rooms to join
```
Comment (meetmangukiya) [inline config.py:26]: Not sure how to use an array in an env var :thinking:

Comment (nemani) [reply config.py:26]: I think its fine if we leave it like this?

**Comment by nemani** [comment]
@rultor merge

**Pull request merged**